src/app/dashboard/dashboard.js

```diff
@@ -21,6 +24,8 @@ export class Dashboard {
             .getSensors()
             .subscribe(data => {
                 this.deviceList = data;
+                this.sensors = data.filter(element => !element.executor);
+                this.executors = data.filter(element => element.executor);
```

Comment:
It would be better that you cover 27-28 lines by simple tests. 
And also remove 26, because I suppose this is not needed after refactor